toolkit/components/filewatcher/tests/xpcshell/xpcshell.ini

```diff
@@ -1,6 +1,6 @@
 [DEFAULT]
 head = head.js
-skip-if = os != "win"
+skip-if = os != "win" && os != "linux" && os != "mac"
```

Comment:
@mgieger to comment.

Reply:
@Yoric Our end goal is to make the changes to each of the branches (linux implementation and mac implementation) separately and then merge them into one branch. From the one complete branch we intend to create a Mercurial patch for Firefox. 
 
The annotated line above definitely would need to be changed in a linux only implementation. With our end goal of combining these branches prior to creating a patch, I had intended to leave this line as is, to avoid confusion since changing it now will just end up being a merge conflict (and will need to be reverted back to the above state). 

If you want, I am happy to change it in this PR as well. Please note in the final patch this current state for this line is what we will want.

Reply:
Generally, we prefer smaller patches when it makes sense. It makes it easier for our automated tools to find out regressions and bugs. So, I'd rather land two patches in Gecko, one for Linux and one for macOS, if that's possible.

Reply:
@tegan-lamoureux I wanted to call attention to Yoric's comment above. This may impact the workflow for tonight / going forward a little bit.

Reply:
Thanks both, this is good to know. We'll center further work around that.